### ISSUE TITLE
quantization model failed

### ISSUE THREAD

**Issue by prio158** [tensorflow/tensorflow#53555]
### 1. System information

- Windows10
- TensorFlow 2.7
- 

### 2. Code

YOLOV5 provides  tensorflow int8 optimization failure

```
# keras_model I wrote is no problem 
converter = tf.lite.TFLiteConverter.from_keras_model(keras_model)
converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS]
converter.target_spec.supported_types = [tf.float16]
converter.optimizations = [tf.lite.Optimize.DEFAULT]
if int8:
      dataset = LoadImages(check_dataset(data)['train'], img_size=imgsz, auto=False)  # representative data
      converter.representative_dataset = lambda: representative_dataset_gen(dataset, ncalib)
      converter.target_spec.supported_ops = [tf.lite.OpsSet.TFLITE_BUILTINS_INT8]
      converter.target_spec.supported_types = []
      converter.inference_input_type = tf.uint8  # or tf.int8
      converter.inference_output_type = tf.uint8  # or tf.int8
      converter.experimental_new_quantizer = False
      f = str(file).replace('.pt', '-int8.tflite')

tflite_model = converter.convert()
open(f, "wb").write(tflite_model)
LOGGER.info(f'{prefix} export success, saved as {f} ({file_size(f):.1f} MB)')
```

TensorFlow Lite: export failure: tensorflow/lite/kernels/conv.cc:349 input->dims->data[3] != filter->dims->data[3] (16 != 1)Node number 3 (CONV_2D) failed to prepare.

**Comment by mohantym** [comment]
Hi @prio158 ! Could you please check this[ thread](https://stackoverflow.com/a/64616086/11530462) for answer?

**Comment by prio158** [comment]
> Hi @prio158 ! Could you please check this[ thread](https://stackoverflow.com/a/64616086/11530462) for answer?

thank you!  That's not the answer I'm looking for, Can TensorFlow Lite quantize Depth separable convolution?

**Comment by mohantym** [comment]
Hi @sachinprasadhs ! Could you please look at this issue ?

**Comment by prio158** [comment]
I found that whenever I used block convolution in the model, I got similar errors

**Comment by sachinprasadhs** [comment]
Group convolution for TFLite has not been implemented yet, you can follow [this](https://github.com/tensorflow/tensorflow/issues/40044) issue for more details and some workaround. Thanks!

**Comment by prio158** [comment]
> Group convolution for TFLite has not been implemented yet, you can follow [this](https://github.com/tensorflow/tensorflow/issues/40044) issue for more details and some workaround. Thanks!

Thank you, I replace  "keras.layers.Conv2D(out_channels , groups=out_channels )"  with  "keras.layers.DepthwiseConv2D",  quantization  success !  but the accuracy of the quantized model is 0。

**Comment by sachinprasadhs** [comment]
Refer [this](https://stackoverflow.com/questions/62015923/) issue for incorrect accuracy for the quanitized model.
